app/assets/javascripts/editor.js

```diff
@@ -109,11 +107,21 @@ class Editor {
   //   this.wrap('#######','')
   // }
   // this function is dedicated to Don Blair https://github.com/donblair
-  save() {
-    localStorage.setItem('plots:lastpost', this.textAreaValue);
+  attachSaveListener() {
+    // remove any other existing eventHandler
+    $("textarea").off("input.save"); // input.save is a custom jQuery eventHandler
+    const thisEditor = this; // save a reference to this editor, because inside the eventListener, "this" points to e.target
+    this.textAreaElement.on("input.save", function() {
+      thisEditor.save(thisEditor) 
```

Comment:
This looks weird, I know. Just to reiterate, JavaScript's `this` acts very funny inside of eventHandlers, so that's why I use `const thisEditor = this` as a workaround.